hapi/hedera-protobufs/services/node_create.proto

```diff
@@ -137,4 +137,29 @@ message NodeCreateTransactionBody {
      * This field is REQUIRED and MUST NOT be set to an empty `KeyList`.
      */
     proto.Key admin_key = 7;
+
+    /**
+    * An ALT_BN128 elliptic curve public encryption key (also controlled <br/>
+     * by the node operator) specific to this node's TSS operations.
+     * <p>
+     * The elliptic curve type may change in the future. For example, <br/>
+     * if the Ethereum ecosystem creates precompiles for BLS12_381,
+     * we may switch to that curve.
+     * <p>
+     * This value SHALL be specified according to EIP-196 and EIP-197 standards, <br/>
+     * See https://eips.ethereum.org/EIPS/eip-196#encoding and <br/>
+     * https://eips.ethereum.org/EIPS/eip-197#encoding
```

Comment:
Note also, when we _do_ want a line break, there shouldn't be a space before it.
We should use markdown links where appropriate, it makes text easier to read.
```suggestion
     * This value SHALL be specified according to EIP-196 and EIP-197 standards,
     * See [EIP-196](https://eips.ethereum.org/EIPS/eip-196#encoding) and
     * (EIP-197](https://eips.ethereum.org/EIPS/eip-197#encoding)<br/>
```